.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       matrix:
         NXF_VER:
-          - "22.10.4"
+          - "25.04.6"
           - "latest-everything"
```

Comment:
curious: is this a real release tag? what's the purpose of running the pipeline against 2 versions of NF in this way?